.github/workflows/build.yml

```diff
@@ -23,12 +21,4 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - run: npm ci
-      - run: node_modules/.bin/gulp lint
       - run: node_modules/.bin/gulp bundle
```

Comment:
I'd swap this around. `gulp lint` is still useful, `gulp bundle` not so much.

Reply:
bundle includes the lint step, and I thought why not run the whole bundling step? it just takes 3 seconds

Reply:
But I'd be fine with just running the linting too

Reply:
Ah yeah I see now. Fair enough.